scripts/india_nfhs/README.md

```diff
@@ -0,0 +1,13 @@
+# National Family Health Survey - India
```

Comment:
Please include additional information in this README:

- Where the cleaned csvs, mcfs and tmcfs are [(example)](https://github.com/datacommonsorg/data/blob/master/scripts/cdc_prams/README.md#cleaned-data)
- Instructions for importing i.e., what commands to run to get raw data and generate the cleaned csv, mcf, tmcf files [(example)](https://github.com/datacommonsorg/data/tree/master/scripts/who/gho#import-procedure)